pinot-core/src/main/java/org/apache/pinot/core/operator/ExecutionStatistics.java

```diff
@@ -22,8 +22,12 @@
  * The <code>ExecutionStatistics</code> class contains the operator statistics during execution time.
  */
 public class ExecutionStatistics {
+  // The number of documents scanned post filtering.
   private long _numDocsScanned;
+  // The number of doc ids scanned in the filtering phase of the query execution: could be larger than the total doc num
```

Comment:
`The number of entries (single-value entry contains one value, multi-value entry may contain multiple values) scanned ...`